src/strings.js

```diff
@@ -1,25 +1,98 @@
-const sayHello = string => {
-  // your code here
+const sayHello = (string) => {
```

Comment:
same problem here, you wrote code for each specific value rather than generalizing your solution dependent on the parameters.